apps/builder/app/builder/features/style-panel/style-source-section.tsx

```diff
@@ -363,11 +367,27 @@ const reorderStyleSources = (styleSourceIds: StyleSource["id"][]) => {
   );
 };
 
-const renameStyleSource = (id: StyleSource["id"], label: string) => {
+const renameStyleSource = (
+  id: StyleSource["id"],
+  name: string
+): StyleSourceError | undefined => {
+  const styleSources = $styleSources.get();
+  if (name.trim().length === 0) {
+    return { name: "minlength", id };
+  }
+  for (const styleSource of styleSources.values()) {
+    if (
+      styleSource.type === "token" &&
+      styleSource.name === name &&
+      styleSource.id !== id
+    ) {
+      return { name: "duplicate", id };
```

Comment:
"type"? "name" got me confused.

Reply:
i thought its like error.name in new Error

Reply:
renamed